File: module_main/src/main/java/com/phone/module_main/request/MainRequest.java
package com.phone.module_main.request;

import java.util.Map;

import io.reactivex.Observable;
import okhttp3.ResponseBody;
import retrofit2.http.FieldMap;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;

public interface MainRequest {

    @FormUrlEncoded
    @POST("/wenda/v1/question/brow/?iid=10344168417&device_id=36394312781")
    Observable<ResponseBody> getMainData(@FieldMap Map<String, String> bodyParams);
}

File: module_main/src/main/java/com/phone/module_main/request/MainRequest.kt
package com.phone.module_main.request

import io.reactivex.Observable
import okhttp3.ResponseBody
import retrofit2.http.FieldMap
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface MainRequest {

    @FormUrlEncoded
    @POST("/wenda/v1/question/brow/?iid=10344168417&device_id=36394312781")
    fun getMainData(@FieldMap bodyParams: Map<String, String>): Observable<ResponseBody>
}